src/utils/EventDispatcher.js

```diff
@@ -113,6 +119,11 @@ define(function (require, exports, module) {
             }
             eventsList[i].handler = fn;
             this._eventHandlers[eventName].push(eventsList[i]);
+            
+            // Check for suspicious number of listeners being added to one object-event pair
+            if (this._eventHandlers[eventName].length > LEAK_WARNING_THRESHOLD) {
+                console.error("Possible memory leak: " + this._eventHandlers[eventName].length + " '" + eventName + "' listeners attached to", this);
```

Comment:
@MarcelGerber Sorry, I didn't see that anyone had reviewed this (if you `@` tag me I'll get an email).

`console.error()` will automatically generate a stack trace.  I don't feel too strongly about warn vs. error, but error doesn't seem _too_ over the top to me considering that this almost always indicates a serious memory leak (both of the worst performance bugs in Brackets history that I can remember were due to leaking listeners).